### ISSUE TITLE
[a11y] Landmarks - Courses

### ISSUE THREAD

**Issue by tdy** [cs-education/classTranscribe#62]
Courses URL: https://classtranscribe.ncsa.illinois.edu/courses

All elements should be contained in landmarks. This supports assistive technologies in organizing a page’s content for users who can’t see/scan the page’s layout (see [WCAG 2.0 SC 2.4.1 Bypass Blocks][1]).

Can be implemented either with [HTML5][2] (`<main>`, `<nav>`, `<header>`, `<footer>`, etc.) or [ARIA][3] (`role=main`, `role=navigation`, `role=banner`, `role=contentinfo`, etc.). There should at least be a “main” landmark at a minimum.

Related: #56, #59

[1]: https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html
[2]: http://www.w3.org/TR/html5/sections.html
[3]: http://www.w3.org/TR/WCAG20-TECHS/ARIA11
